packages/liferay-npm-scripts/test/format/__snapshots__/formatJSP.js.snap

```diff
@@ -928,7 +928,45 @@ exports[`formatJSP() formatting entire fixtures page.jsp matches snapshot 1`] =
 "
 `;
 
-exports[`formatJSP() formatting entire fixtures recaptcha.jsp matches snapshot 1`] = `null`;
+exports[`formatJSP() formatting entire fixtures recaptcha.jsp matches snapshot 1`] = `
+"<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file=\\"/captcha/init.jsp\\" %>
+
+<c:if test=\\"<%= captchaEnabled %>\\">
+	<script src=\\"<%= HtmlUtil.escapeAttribute(captchaConfiguration.reCaptchaScriptURL()) %>?hl=<%= HtmlUtil.escapeAttribute(locale.getLanguage()) %>\\" type=\\"text/javascript\\"></script>
```

Comment:
Note the reason this script content is included in the snapshot now is that it contains no formattable elements (it has a `<script>` tag, but it has no contents). Previously we `formatJSP` would return `null` to indicate nothing formattable, but in this PR it returns the original source, unmodified.

Same deal with the other snapshot further down.